docs/tutorials/cargo_tests/baked/src/main.rs

```diff
@@ -0,0 +1,24 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+//! This is a demo project using baked data.
+//! 
+//! For more information, see the tutorial [cargo.md](../../cargo.md).
+
+use icu::locid::locale;
+use icu::plurals::PluralCategory;
+use icu::plurals::PluralRules;
+
+mod baked {
+    include!("../baked_data/mod.rs");
```

Comment:
Use the new style of declaring a struct and using the macro to add the impls.

Reply:
Can't do that until the new datagen is released. These examples are on crates.io code versions.

Reply:
.

Reply:
This is done.

Reply:
* I prefer defining the struct one level above the `include`
* I realize we're doing this in testdata to align with the constructors, but I'd rather call it `BakedProvider` than `UnstableProvider`. There's nothing unstable about the provider itself.

Reply:
done